### DISCUSSION TITLE
L:ocal PDF files not displayed. Question about versions.

### DISCUSSION THREAD

**Discussion by krubin723** [cefsharp/CefSharp#4633] [General]
Hi,

We had an issue with using some of our features in a CRefSharp desktop application and updated from version 75 to 99, We use Windows 10 PCs Standard and Enterprise. 
After upgrading to version 99, we find that our Wpf application cannot popup a print dialog box that allows us to print the local pdf file.
Can you please tell me the latest version of CrefShaRP that is able to popup this print dialog box?

I've seen where developers create a custom request handler, but for now I would (and my management) like to know what the last version of CrefShaRP that is able to popup this print dialog box?.

Thank you,
Ken

**Comment by amaitland** [comment]
`M119` is the current supported version. https://github.com/cefsharp/CefSharp.MinimalExample is kept up to date for testing purposes.

If you can reproduce the problem using M119 then please provide more information about **how** you are loading the PDF. Please make sure to include a code example.

> After upgrading to version 99

Not supported. Upgrade to M119
